### PR TITLE
Feature/dev 367 btc input

### PR DESCRIPTION
**Pull request by ThaBrad**
#### What kind of PR is this?:
/kind feature

#### Additional comments?:
Only for Beta User still??

#### Developer Checklist:
- [x] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx [hunk 1]
```diff
@@ -190,6 +192,20 @@ export function PortfolioNavigator (): JSX.Element {
         }}
       />
 
+      <PortfolioStack.Screen
+        component={ReceiveDTokenScreen}
+        name='ReceiveDTokenScreen'
+        options={{
+          headerTitle: () => (
+            <HeaderTitle
+              text={translate('screens/ReceiveDTokenScreen', 'Bitcoin Deposit')}
```
Comment (davidleomay) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx:201]: DToken or Bitcoin?

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx [hunk 2]
```diff
@@ -84,104 +94,91 @@ export function DfxButtons (): JSX.Element {
...
       }
     },
     {
-      img: {
-        en: BtnOverview
-      },
+      Svg: BtcIcon,
+      label: translate('screens/DfxButtons', 'Deposit Bitcoin'),
+      onPress: () => {
+        // check kycData
```
Comment (davidleomay) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx:115]: ?

File: shared/translations/languages/pt.txt [hunk 3]
```diff
@@ -1256,7 +1255,6 @@
     "Available to sell": "Disponível para vender",
     "Fee": "Taxa",
     "Each transaction will be subject to a small amount of fees.": "Jede Transaktion unterliegt einer geringen Gebühr.",
-    "Transfer to your bank account": "",
```
Comment (Krysh90) [inline shared/translations/languages/pt.txt:1259]: Why is pt a txt file and why are those removed? cleanup?

File: shared/translations/languages/es.json [hunk 4]
```diff
@@ -1569,5 +1569,11 @@
     "Wallet created!": "Cartera creada!",
     "Access decentralized finance with Bitcoin-grade security, strength and immutability.": "Accede a finanzas descentralizadas con el grado de seguridad, fuerza e inmutabilidad de Bitcoin.",
     "Continue": "Continúa"
+  },
+  "screens/ReceiveDTokenScreen": {
+    "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!": "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!",
```
Comment (Krysh90) [inline shared/translations/languages/es.json:1574]: other languages like fr and it don't have this line

File: shared/translations/languages/de.json [hunk 5]
```diff
@@ -1567,5 +1567,19 @@
     "Wallet created!": "Wallet erstellt!",
     "Access decentralized finance with Bitcoin-grade security, strength and immutability.": "Zugang zu dezentralem Finanzwesen mit Sicherheit, Stärke und Unveränderlichkeit auf Bitcoin-Niveau.",
     "Continue": "Weiter"
+  },
+  "screens/ReceiveDTokenScreen": {
+    "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!": "Mindest… {{MinimumBtcAmount}}!",
+    "DEPOSIT ADDRESS": "EINZAHL ADRESSE",
+    "Send ONLY Bitcoin (BTC) to your BTC deposit address shown below or scan the QR code. We will transfer dBTC in your DFX Wallet afterwards.": "Sende NUR …… Bitcoin (BTC) to your BTC deposit address shown below or scan the QR code. We will transfer dBTC in your DFX Wallet afterwards.",
```
Comment (Krysh90) [inline shared/translations/languages/de.json:1574]: mixture between german and english

File: shared/translations/languages/de.json [hunk 6]
```diff
@@ -1567,5 +1567,19 @@
     "Wallet created!": "Wallet erstellt!",
     "Access decentralized finance with Bitcoin-grade security, strength and immutability.": "Zugang zu dezentralem Finanzwesen mit Sicherheit, Stärke und Unveränderlichkeit auf Bitcoin-Niveau.",
     "Continue": "Weiter"
+  },
+  "screens/ReceiveDTokenScreen": {
+    "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!": "Mindest… {{MinimumBtcAmount}}!",
```
Comment (Krysh90) [inline shared/translations/languages/de.json:1572]: Is this the wanted translation for that key?

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx [hunk 7]
```diff
@@ -84,104 +94,91 @@ export function DfxButtons (): JSX.Element {
...
     {
-      img: {
-        de: BtnDfxDe,
-        en: BtnDfxEn,
-        fr: BtnDfxFr,
-        it: BtnDfxIt,
-        es: BtnDfxEs
-      },
+      Svg: DfxIcon,
+      label: translate('screens/DfxButtons', 'Buy & Staking'),
```
Comment (Krysh90) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx:100]: nice changes 👍

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+  }
+}
+
+export enum CryptoButtonGroupTabKey {
+  DFI = 'DeFiChain',
+  BTC = 'Bitcoin',
+  ETH = 'Ethereum'
+}
+
+export const MinimumBtcAmount = '0.0005 BTC'
```
Comment (Krysh90) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx:45]: maybe we should add a comment/todo here, that we are using API values as soon as API delivers those.

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+}: Props): JSX.Element {
+  const logger = useLogger()
+  const { isLight } = useThemeContext()
+  const { address } = useWalletContext()
+  const [activeButton, setActiveButton] = useState<CryptoButtonGroupTabKey>(route.params?.crypto ?? CryptoButtonGroupTabKey.DFI)
+
+  const [bitcoinAddress, setBitcoinAddress] = useState('')
+  const activeAddress = activeButton === CryptoButtonGroupTabKey.DFI ? address : bitcoinAddress
+  const [isLoading, setIsLoading] = useState(false)
+  const defaultFee = 1.2
```
Comment (Krysh90) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx:61]: fee is already being delivered through the route objects

File: shared/translations/languages/pt.txt (hunk 3 above)
Comment (ThaBrad) [reply shared/translations/languages/pt.txt:1259]: temporarily changed to disable

File: shared/translations/languages/es.json (hunk 4 above)
Comment (ThaBrad) [reply shared/translations/languages/es.json:1574]: just stubs, forwarded to translation team

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx (hunk 9 above)
Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ReceiveDTokenScreen.tsx:61]: is loaded, just a fallback to make **sure** a fee is shown

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx (hunk 1 above)
Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx:201]: Screen Design just says Bitcoin Deposit. Component will grow in future, therefore future ready naming. Will change when more external Crypto's are added.

File: metro.config.js [hunk 10]
```diff
@@ -3,4 +3,24 @@ const defaultConfig = getDefaultConfig(__dirname)
 
 defaultConfig.resolver.extraNodeModules.stream = require.resolve('stream-browserify')
 
-module.exports = defaultConfig
+// module.exports = defaultConfig
```
[the remark this answers is not in the document]
Comment (ThaBrad) [reply metro.config.js:6]: Left it in, in case I have to refactor bc of broken RN-web support with SVG update :-|

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx (hunk 2 above)
Comment (ThaBrad) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DfxButtons.tsx:115]: extended comment
